<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NuevoRegistro</class>
 <widget class="QDialog" name="Dialog_NuevoRegistro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GB_NuevoRegistro">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>551</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Nuevo registro</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>551</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QLineEdit" name="LE_CodPostalNew"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="LE_TelefonoNew"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Correo electrónico</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_DireccionNew"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LE_NombreNew"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="LE_EmailNew"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Código postal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LE_PoblacionNew"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Dirección</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Población</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn_GuardarRegistro">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar registro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
